pkg/rc/replication_controller.go

```diff
@@ -396,7 +401,8 @@ func (rc *replicationController) ensureConsistency(current types.PodLocations) e
 		}
 
 		rc.logger.WithField("node", pod.Node).WithField("intentManifestSHA", intentSHA).Info("Found inconsistency in scheduled manifest")
-		if err := rc.schedule(ctx, pod.Node); err != nil {
+
+		if err := rc.scheduleNoAudit(ctx, pod.Node); err != nil {
```

Comment:
actually we need that `Context()` getter unless we're going to move node scheduling logic into the auditing transaction as well. i think things are fine the way they are